Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -308,5 +294,21 @@ private static string ToClassName(this IntPtr hwnd)
             User32.GetClassName(hwnd, name, name.Capacity);
             return name.ToString();
         }
+
+        [Conditional("THIRSTY_DUCK")]
+        [Conditional("THIRSTY_DUCK_EVT")]
```

Comment:
Does that make it `&&` or `||`?

Reply:
That's a `||` - `THIRSTY_DUCK_EVT` or `THIRSTY_DUCK_WM` gives you a subset of message pump from different parts; `THRISTY_DUCK` gives you the both.